src/test/storage/iterables_test.cpp

```diff
@@ -247,11 +308,11 @@ TEST_F(IterablesTest, ReferenceSegmentIteratorWithIterators) {
 
   auto iterable = ReferenceSegmentIterable<int, EraseReferencedSegmentType::No>{*reference_segment};
 
-  auto sum = uint32_t{0};
+  auto sum = int{0};
```

Comment:
Why using the sum as an integer when you are going to compare against an unsigned value in EXPECT_EQ?

Reply:
Yep, better change that to signed. And I'd also vote for `int32_t` which is the exact type behind our "int" (see all_type_variant.hpp)